packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx

```diff
@@ -61,6 +63,20 @@ function NavLink({
 }
 
 function NavItemDesktop({items, position, className, ...props}) {
+  const dropDownRef = React.useRef<HTMLDivElement>(null);
+  const dropDownMenuRef = React.useRef<HTMLUListElement | any>(null); // TODO should find better solution for this. anything else will get a error when retrieve children.
```

Comment:
I'd rather not have the `| any` here

Prefer // @ts-expect-error on the focus call, but this is surprising it would be required